examples/63_Conic.html

```diff
@@ -36,12 +37,45 @@
 		              {name:"Co", type:"ConicBy5", args:["A","B","C","D","E"],color:[0,0,1],alpha:1,size:1},
 //		     {name:"l", type:"Join", args:["A", "D"]}
                       ];
-            createCindy({canvasname:"CSCanvas",
+            var cdy=createCindy({canvasname:"CSCanvas",
                         movescript:"csmove",
                         grid:1,
                         snap:true,
                         geometry:gslp});
 
+            var cases = [
+                ["parabola1", [0,0,1], [1,1,1], [2,4,1], [-2,4,1], [-1,1,1]],
```

Comment:
@gagern:The instrumentation was added entirely within `eval_helper.drawconic = function(conicMatrix, modifs)` next to any `csctx` calls:
```javascript
            console.log("lineTo(" + [pt2.px, pt2.py] + ")");
            return csctx.lineTo(pt2.px, pt2.py);
...
            console.log("lineTo(" + [pt2.px, pt2.py] + ")");
            return csctx.lineTo(pt2.px, pt2.py);
...
        console.log("quadraticCurveTo(" + [cx, cy, pt2.px, pt2.py] + ")");
        csctx.quadraticCurveTo(cx, cy, pt2.px, pt2.py);
...
    csctx.beginPath();
    console.log("beginPath(" + starts.length + ", " + specialPoints.length + ")");
...
        csctx.moveTo(pt.px, pt.py);
        console.log("moveTo(" + [pt.px, pt.py] + ")");
...
                csctx.closePath();
                console.log("closePath()");
...
    csctx.stroke();
    console.log("stroke(" + starts.length + ", " + specialPoints.length + ")");
```

The console log shows:
```
beginPath(13, 8)
moveTo(157.0254045208004,0)
quadraticCurveTo(250.50000000000003,699.0000000000002,343.9745954791996,0)
stroke(13, 8)
```
Applying a 45&deg; rotation matrix to the those control points would give the parabola2 (although not clipped correctly).